entries/f/facebook.com.json

```diff
@@ -1,6 +1,9 @@
 {
   "Facebook": {
     "domain": "facebook.com",
+    "additional-domains": [
+      "messenger.com"
+    ],
```

Comment:
I'm not quite sure about this one. It seems to deserve a separate entry in `communication`.

Reply:
I'm not sure I agree here. As Messanger is a part of the Facebook experience and can't be severed from it I don't think a separate entry is needed for it. It's basically the PM feature of Facebook like the DMs on Twitter or private messages on Discord.

Reply:
I believe this should just be give added as an addition domain. As said before it's basically like the pm feature on twitter.